bin/setup-dev-env

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+# sets up a linux system (e.g. a virtualbox VM managed by Vagrant) to be a dev environment
+
+ag () {
+    sudo DEBIAN_FRONTEND=noninteractive apt-get -yqq --no-install-recommends ${@}
+}
+
+sql () {
+    sudo -u postgres psql -d postgres -c "${@}"
+}
+
+# add node repo
+curl --silent https://deb.nodesource.com/gpgkey/nodesource.gpg.key | sudo apt-key add -
+echo "deb https://deb.nodesource.com/node_7.x xenial main" | sudo tee /etc/apt/sources.list.d/nodesource.list
```

Comment:
Oh, I meant to roll that back.  I'll fix it ASAP.  I was fighting a super irritating node-gyp issue and thought going forward might silence it.